deployment/ccip/changeset/deploy.go

```diff
@@ -67,25 +63,35 @@ var (
 	USDCTokenPool       deployment.ContractType = "USDCTokenPool"
 )
 
-func DeployPrerequisiteChainContracts(e deployment.Environment, ab deployment.AddressBook, selectors []uint64) error {
+func DeployPrerequisiteChainContracts(e deployment.Environment, ab deployment.AddressBook, selectors []uint64, usdcEnabledChains []uint64) error {
```

Comment:
lets make this function private to enforce that external packages (integration-tests, chainlink-deployment etc.) only use the changeset interface